OpenSourcePolitics/decidim-imt
# Fix document import

#### :tophat: Description
When importing a participatory text, the title of article is now "paragraph number_of_paragraph", and it is possible to delete an article.

#### :pushpin: Related Issues

- [Notion card](https://www.notion.so/opensourcepolitics/bd4bf860eab94f9daca7da803f09e3b2?v=c96df65755dd403d95e86f82d7749709&p=c9bc935e3371405b9fe4b4047f1867eb&pm=c)

#### Testing

1. As an admin, go to one process, add a new proposal component and configure it with "Participatory texts enabled".
2. Go to the proposal component and click on "participatory texts" button, and then on "import document".
3. Click on markdown (cf screenshot), get the uploaded .md file
4. Click on add file and upload the .md file, click on "save" and then on "upload document".
5. See that all the titles of articles are named "paragraph 1, paragraph 2..." instead of "1,2..."
6. Delete article with paragraph 3 title, and see that the article is deleted and titles of articles after the one deleted are updated so it doesn't miss a number.

#### Tasks
- [ X] Add specs


### :camera: Screenshots
<img width="1046" alt="Capture d’écran 2024-08-20 à 13 38 20" src="https://github.com/user-attachments/assets/ef76d04d-7014-4ecd-b77f-fa77c17fd85d">
d to the UI*
